resources/atlantis_installer.rb

```diff
@@ -20,13 +19,18 @@
   # install atlantis
   ark 'atlantis' do
```

Comment:
I'm thinking putting atlantis and terraform in /opt as follows: `ark 'atlantis' do
  url <<url>>
  version <<version>>
  checksum <<checksum>>
  prefix_root '/opt'
  prefix_home '/opt'
  has_binaries ['atlantis']
  strip_components 0
  owner <<user>>
  group <<group>>
end`


This way we can retain multiple versions if we want to. Also, with has_binaries, ark will automatically symlink to /usr/local/bin/